test/core/client_channel/lb_policy/lb_policy_test_lib.h

```diff
@@ -160,10 +160,33 @@ class LoadBalancingPolicyTest : public ::testing::Test {
 
     const std::string& address() const { return address_; }
 
+    void AssertValidConnectivityStateTransition(
+        grpc_connectivity_state to_state, grpc_connectivity_state from_state,
```

Comment:
Suggest swapping the order of these two parameters, since that seems a little more intuitive (old state followed by new state).